modules/serverless-logic-upgrade-1-36-finalizing-data-index.adoc

```diff
@@ -0,0 +1,72 @@
+
+.Procedure
+
+. List all the triggers by running the following command:
++
+[source,terminal]
+----
+$ oc get triggers -n <target_namespace>
+----
++
+.Example output
+[source,sql,subs="verbatim,quotes"]
+----
+NAME                                                              BROKER              SUBSCRIBER_URI
+data-index-jobs-a25c8405-f740-47d2-a9a5-f80ccaec2955              example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/jobs
+data-index-process-definition-473e1ddbb3ca1d62768187eb80de99bca   example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/definitions
+data-index-process-error-a25c8405-f740-47d2-a9a5-f80ccaec2955     example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-instance-mul07f593476e8c14353a337590e0bfd5ae   example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-node-a25c8405-f740-47d2-a9a5-f80ccaec2955      example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-state-a25c8405-f740-47d2-a9a5-f80ccaec2955     example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+data-index-process-variable-487e9a6777fff650e60097c9e17111aea25   example-broker      http://sonataflow-platform-data-index-service.<target_namespace>.svc.cluster.local/processes
+
+jobs-service-create-job-a25c8405-f740-47d2-a9a5-f80ccaec2955      example-broker      http://sonataflow-platform-jobs-service.<target_namespace>.svc.cluster.local/v2/jobs/events
+jobs-service-delete-job-a25c8405-f740-47d2-a9a5-f80ccaec2955      example-broker      http://sonataflow-platform-jobs-service.<target_namespace>.svc.cluster.local/v2/jobs/events
+----
+
+. Based on the generated example output, delete the old `data-index-process-definition` trigger by running the following command:
++
+[source,terminal]
+----
+$ oc delete trigger data-index-process-definition-473e1ddbb3ca1d62768187eb80de99bca -n <target_namespace>
+----
++
+After deletion, a new trigger compatible with {ServerlessLogicProductName} 1.36.0 is automatically created.
+
+. (Optional) Identify the old `ReplicaSet` resource by running the following command:
++
+[source,terminal]
+----
+$ oc get replicasets -o custom-columns=Name:metadata.name,Image:spec.template.spec.containers[*].image -n <target_namespace>
+----
++
+.Example output
+[source,sql,subs="verbatim,quotes"]
+----
+Name                                                Image
+*sonataflow-platform-data-index-service-1111111111   registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8:1.35.0*
```

Comment:
I'd remove this bolding, we don't do that on OCP docs as far as I know, and I don't think it's even working. Same for the other place you do it in another file.